LondonBritishLibrary/orient/BLorient674.xml

```diff
@@ -132,197 +132,202 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     </item>
                                     
                                     <item xml:id="a3">
-                                        <locus target="#87rbc"/>
-                                        <desc type="ReceiptNote">Two Amharic receipt notes or
-                                            inventories of liturgical items and books, mentioning <persName ref="">ʾAzzāž Takla Hāymānot</persName> and 
-                                            <persName ref="">Blāttǝn Getā Takkā</persName>, <persName ref="">Walda Dāwit</persName>.</desc>
+                                        <locus target="#87rb"/>
+                                        <desc type="ReceiptNote">Amharic receipt note or
+                                            inventory of liturgical items and books, mentioning <persName ref="PRS13699TaklaH">ʾAzzāž Takla Hāymānot</persName> and 
```

Comment:
could be also PRS9154TaklaHa?

Reply:
The EAe dates him to the 16th c., so the identification seemed unlikely to me